### ISSUE TITLE
Updated .js not refreshing in simulator if error

### ISSUE THREAD

**Issue by ExpoNewPerson** [expo/expo-cli#304]
I have a basic app, which opens and launches fine through Expo, Metro Bundler and opens simulator and looks fine.

If I make error in syntax and try and run Simulator it gives me the error, as expected.

Syntax error here;

  46 |          <View>
  47 |       <Image
> 48 |           <Image source={require('../assets/images/icons/ic_clear_black_48dp.png')} />
     |           ^
  49 |         </View>

But then I change the file and re run the Simulator (even closing and opening the Simulator) the new file change is not picked up, I get the same error. It's like the SIM is caching the file ...

Here's what I changed the Syntax to, but still get error referring to above.

<View>
      <Image source={require('../assets/images/icons/ic_clear_black_48dp.png')} />
        </View>

Any help greatly appreciated. Thank you.

**Comment by tcdavis** [comment]
Have you tried restarting expo-cli instead of the simulator?

**Comment by ExpoNewPerson** [comment]
Thanks so much for replying. That was what I ended up having to do, restart everything anytime I wanted to check the build (ie whenever I typed anything).

But actually I figured the Sim was not updating because something was blocking it - and it ended up being a very full disc. Everything was working OK, but I think the SIM takes a lot of juice, so I cleared some and hey presto, back to normal.

Thanks so much again for the response.
